recipe/meta.yaml

```diff
@@ -1,24 +1,27 @@
 {% set name = "openvino" %}
 {% set version_postfix = "2023.1.0" %}
-{% set version = "2023.1.0.dev20230623" %}
+{% set version = "2023.1.0.dev20230728" %}
 
 package:
   name: {{ name|lower }}-split
   version: {{ version }}
 
 source:
-  - url: https://github.com/openvinotoolkit/openvino/archive/26d53eb1da3cadc3a43562ab8a655e00919e6689.tar.gz
-    sha256: 21130f197d25ad5f361e4cce705ae16eda2b48c8dc55417bb379fc63d9b6366b
+  - url: https://github.com/openvinotoolkit/openvino/archive/c7cde6a102a98f1e66d06f064915eaccd97b7c99.tar.gz
+    sha256: 78e9eb0661463c667d82da64a66b6823cf718b47f4778795159fa88cf7d87dce
     patches:
     - patches/002-win32-python-search.patch
     - patches/003-lin-arm64-compute-lib.patch  # [aarch64]
     - patches/004-gflags-samples.patch
   - url: https://github.com/ARM-software/ComputeLibrary/archive/refs/tags/v23.02.1.tar.gz  # [aarch64 or arm64]
     sha256: c3a443e26539f866969242e690cf0651ef629149741ee18732f954c734da6763  # [aarch64 or arm64]
     folder: src/plugins/intel_cpu/thirdparty/ComputeLibrary  # [aarch64 or arm64]
-  - url: https://github.com/openvinotoolkit/oneDNN/archive/f1080be59e8eaa053195cd06da833f6515e29a96.tar.gz
-    sha256: dfac2fc8a728ec8f5310e5144c3e4be547752bdcc45db1461249a73c5ff245a4
+  - url: https://github.com/openvinotoolkit/oneDNN/archive/5bf389f616d0b1551ce468f2712df492a7aab140.tar.gz
+    sha256: 31af79c5b25a01ae795ad03e0745d238f0459b6e79844c33f5cf8b549d005393
     folder: src/plugins/intel_cpu/thirdparty/onednn
+  - url: https://github.com/openvinotoolkit/mlas/archive/519abf79de5ee295cfe5bbed97037a2623616c80.tar.gz
+    sha256: ef681b16a0f8b325aa01ab94d252f4cbd2f4848d8a1f8a3f2bd12ee76b241238
+    folder: src/plugins/intel_cpu/thirdparty/mlas
```

Comment:
let's add `win64 or linux64`

Reply:
But it also failed on OSX

Reply:
Oh, right. 
This option is available only for x86_64 https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/CMakeLists.txt#L37, which implies arm64 as well.

Reply:
I see now it's for x86_64
https://github.com/openvinotoolkit/openvino/blob/5572fad7129b2ae859cd167aa3365f4caa2fb624/src/plugins/intel_cpu/CMakeLists.txt#L37